scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/MerlInsightRulesTest.java

```diff
@@ -51,11 +52,49 @@ public void schedule(){
   public void firstRule() {
     smallProblem.setGoals(new ArrayList<>(rules.getFirstRuleGoals().values()));
     schedule();
+    var time = planningHorizon.getStartAerie().plus(Duration.MINUTE);
+    assert(TestUtility.activityStartingAtTime(plan, time, rules.getActivityType("SSAMonitoring")));
+    assert(TestUtility.activityStartingAtTime(plan, time, rules.getActivityType("HP3TemP")));
+    for(var t = Duration.HOUR; t.shorterThan(planningHorizon.getEndAerie()); t = t.plus(Duration.HOUR)){
+      assert(TestUtility.activityStartingAtTime(plan, time, rules.getActivityType("HP3TemP")));
```

Comment:
I think we want to use JUnit's `assertTrue` rather than the built-in Java assert. That way we get nice test failure messages. You can get that by adding this import statement:

```
import static org.junit.jupiter.api.Assertions.*;
```

and then changing all the `assert`s in this file to `assertTrue`

Reply:
(but also, `assert` does nothing by default unless you run the JVM with the `-ea` flag)

Reply:
I have replaced all assert by assertTrue or assertEquals and so on in all the scheduler tests.